src/client/components/nav/style.scss

```diff
@@ -1,48 +1,45 @@
 @import "../../styles/variables";
 
+$text-color: rgba(255, 255, 255, 0.7);
+
+$icons: (
+  project: "../../assets/project.svg",
+  version: "../../assets/version.svg",
+  makers: "../../assets/team.svg",
+  snippets: "../../assets/snippets.svg"
+);
+
 .nav {
-  display: flex;
-  align-items: center;
-  ul {
-    display: flex;
-    width: 100%;
+  display: grid;
+  grid-gap: 0 0.5rem;
+  grid-template-columns: minmax(min-content, max-content) 50px 1fr 80px;
+  padding: 0 1rem;
+  width: 100%;
+  background: $darker;
+  border-radius: 16px 16px 0 0;
+
+  li {
+    color: $text-color;
+    font-weight: bold;
+    font-size: 12px;
+    text-transform: uppercase;
+  }
+
+  &-link {
+    color: $text-color;
     padding-left: 5px;
-    li {
-      padding-left: 24px;
-      margin-right: 20px;
-      background-position: left center;
-      background-repeat: no-repeat;
-      background-size: 16px;
-      color: rgba(255, 255, 255, 0.7);
-      font-weight: bold;
-      font-size: 12px;
-      text-transform: uppercase;
-      &.project {
-        background-image: url("../../assets/project.svg");
-      }
-      &.version {
-        background-image: url("../../assets/version.svg");
-      }
-      &.makers {
-        background-image: url("../../assets/team.svg");
-      }
-      &.repo {
```

Comment:
I didn't see this class being used anywhere, didn't add it to the `$icons`

Reply:
actually that was supposed be repo URL but not needed for now!